srcpkgs/trash-cli/template

```diff
@@ -1,14 +1,20 @@
 # Template file for 'trash-cli'
 pkgname=trash-cli
-version=0.20.12.26
+version=0.21.7.24
 revision=1
 build_style=python3-module
-hostmakedepends="python3"
+make_check_args="--deselect='tests/test_files.py::Test_make_unreadable_file' \
+ --deselect='tests/test_files.py::Test_make_unreadable_dir' \
+ --deselect='tests/test_trash_empty.py::TestTrashEmptyCmd' \
+ --deselect='tests/test_trash_list.py::Test_describe_trash_list' \
+ --deselect='tests/test_files.py::Test_make_unreadable_dir' \
+ --deselect='tests/test_trash_empty.py::TestTrashEmptyCmd'" # broken in CI
```

Comment:
There are still broken tests; I'd consider doing `make_check=ci-skip` and removing `make_check_args`, if the tests run successfully locally.